Resfeber/View Controllers/MainTabBarController.swift

```diff
@@ -17,5 +39,56 @@ class MainTabBarController: UITabBarController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         UITabBar.appearance().tintColor = UIColor.Resfeber.red
+        configureViewControllers()
+    }
+    
+    // MARK: - Selectors
+
+    @objc fileprivate func profileImageTapped() {
+        print("DEBUG: profileImageTapped..")
+    }
+
+    // MARK: - Helpers
+    
+    private func configureViewControllers() {
+        let symbolConfig = UIImage.SymbolConfiguration(weight: .bold)
+        
+        let exploreVC = ExploreViewController()
+        let exploreTabImage = UIImage(systemName: "house", withConfiguration: symbolConfig)
+        let exploreNav = tabNavigationController(rootViewController: exploreVC,
+                                                 tabImage: exploreTabImage,
+                                                 navBarTitle: "Explore")
+        
+        let favoritesVC = FavoritesViewController()
+        let favoritesTabImage = UIImage(systemName: "heart", withConfiguration: symbolConfig)
+        let favoritesNav = tabNavigationController(rootViewController: favoritesVC,
+                                                   tabImage: favoritesTabImage,
+                                                   navBarTitle: "Favorites")
+        
+        let itineraryVC = ItineraryViewController()
+        let itineraryTabImage = UIImage(systemName: "briefcase", withConfiguration: symbolConfig)
+        let itineraryNav = tabNavigationController(rootViewController: itineraryVC,
+                                                   tabImage: itineraryTabImage,
+                                                   navBarTitle: "Itinerary")
+        
+        viewControllers = [exploreNav, favoritesNav, itineraryNav]
+    }
+    
+    private func tabNavigationController(rootViewController: UIViewController, tabImage: UIImage?, navBarTitle: String? = nil) -> UINavigationController {
```

Comment:
I'd consider adding documentation to some of these functions to memorialize what it does (and potentially re-use in other code): select function and  ⌥+⌘+/ is the keyboard shortcut 

```
/// <#Description#>
/// - Parameters:
///   - rootViewController: <#rootViewController description#>
///   - tabImage: <#tabImage description#>
///   - navBarTitle: <#navBarTitle description#>
/// - Returns: <#description#>
```